src/components/Faq.content.tsx

```diff
@@ -66,9 +98,20 @@ export const faqQuestionsAnswers = [
 		question: 'How do I know how big a file is?',
 		answer: (
 			<p>
-				In ArDrive, before any files are uploaded to the network, we calculate their size and estimate the price
-				to upload. Before it is paid for and stored permanently, you must approve the upload. This ensures you
-				are only uploading the right data for the right price.
+				In ArDrive, before any files are uploaded to the network, we
+				calculate their size and estimate the price to upload. Before it
+				is paid for and stored permanently, you must approve the upload.
+				This ensures you are only uploading the right data for the right
+				price.
+			</p>
+		)
+	},
+	{
+		question: 'How do I use a credit card?',
+		answer: (
+			<p>
+				You can purchase Credits with your credit card in the web app
```

Comment:
Check that Credits should be capitalized; I don't think it should be personally but would like this double-checked.

Reply:
I did a copy/paste, but I can check this with Ali.

Reply:
She confirmed it should be capitalized, yes 💯